docs/markdown/ios/overlaypanel.md

```diff
@@ -0,0 +1,116 @@
+---
```

Comment:
I assume this is basically the same content as Android, so same comments as from that file

Reply:
UNDOING